### ISSUE TITLE
INFO - User Interface — Report Readability Umbrella

### ISSUE THREAD

**Issue by ericthomasswenson** [Equipment-and-Tool-Institute/j1939-84#1102]
Document a list of User Interface / Report Readability Suggestions. 

15 Suggestions at present need to be scheduled as 
Task 5 Release, 
Task 8 candidate, or 
future maintenance.

**Comment by ericthomasswenson** [comment]
#1037 Add version number display to User Interface Data Collection Dialog Box 
Jun 7, 2021 by ericthomasswenson

#1047 Display FF length from DM24 
Jul 7, 2021 by ericthomasswenson

#857 Increase text font size in pop up dialogue boxes 
Mar 22, 2021 by ericthomasswenson
  
#855 Increase pop up window font size. 
Mar 22, 2021 by ericthomasswenson

#854 Increase Font Size in Status Bar future enhancement
Mar 22, 2021 by ericthomasswenson

#832 Test 7.3 & 7.15 Spacing future enhancement …. Deferred to Task 8
Mar 22, 2021 by mmschaefer
  
#827 Beep When User Response Required 
Mar 22, 2021 by mmschaefer

#1038 SPN target missing 
Jun 8, 2021 by ericthomasswenson

#1024 Correct grammatical error in 6.8.10.3. 
Jun 4, 2021 by ericthomasswenson

#984 Add on screen timer for engine running time 
May 6, 2021 by gumbelmj

#1008 Update help file graphics to show selected baud rate where omitted 
May 26, 2021 by ericthomasswenson  …. Deferred to Task 8

#656 “SPN" should likely be changed to "SP" in most cases 
Feb 26, 2021 by gumbelmj   …. Deferred to Task 8

#958 There should be means to continue the test when failure implants fail 
Apr 22, 2021 by ericthomasswenson

#804 "Waiting" dialogs should include a timer so the operator knows how long they've waited 
Mar 19, 2021 by gumbelmj

#803 Key state dialogs should be removed once the action is complete 
Mar 19, 2021 by gumbelmj

**Comment by ericthomasswenson** [comment]
Disposition by users to Task 5 and Task 8 complete 4/15/2022
Referenced GitHub issues have Task 5 and Task 8 tags.
Close when Task 5 release report plan is complete.

**Comment by ericthomasswenson** [comment]
Items ready for Task 5 assigned to Joe.
